Godeps/_workspace/src/k8s.io/kubernetes/pkg/genericapiserver/server_run_options.go

```diff
@@ -46,6 +46,7 @@ func NewServerRunOptions() *ServerRunOptions {
 		InsecureBindAddress:  net.ParseIP("127.0.0.1"),
 		InsecurePort:         8080,
 		LongRunningRequestRE: defaultLongRunningRequestRE,
+		MaxRequestsInFlight:  400,
```

Comment:
Also, Kube doesn't always do their options "right" so we don't get the tap
on the shoulder in the code where the new options are obviously missing.

On Wed, Apr 6, 2016 at 6:19 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In
> Godeps/_workspace/src/k8s.io/kubernetes/pkg/genericapiserver/server_run_options.go
> <https://github.com/openshift/origin/pull/8395#discussion_r58792210>:
>
> > @@ -46,6 +46,7 @@ func NewServerRunOptions() *ServerRunOptions {
> >  		InsecureBindAddress:  net.ParseIP("127.0.0.1"),
> >  		InsecurePort:         8080,
> >  		LongRunningRequestRE: defaultLongRunningRequestRE,
> > +		MaxRequestsInFlight:  400,
>
> This is per master
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8395/files/714b88b4308eb44f74ead7e66395fd98aa98207c#r58792210>
>